kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProtocolFilterReconciler.java

```diff
@@ -176,7 +176,9 @@ public UpdateControl<KafkaProtocolFilter> reconcile(
         KafkaProtocolFilter newFilter = newFilterWithCondition(filter, conditionBuilder.build());
         LOGGER.debug("Patching with status {}", newFilter.getStatus());
         UpdateControl<KafkaProtocolFilter> uc = UpdateControl.patchStatus(newFilter);
-        LOGGER.info("Completed reconciliation of {}/{}", namespace(filter), name(filter));
+        if (LOGGER.isInfoEnabled()) {
```

Comment:
Is sonar really whinging about stuff logged at info?!

Reply:
It was indeed.